src/services/local-storage.js

```diff
@@ -4,8 +4,9 @@ import { useState, useEffect } from "react";
 
 function getValue(key, defaultValue) {
   try {
-    return JSON.parse(localStorage.getItem(key)) || defaultValue;
-  } catch(e) {
+    const storedValue = JSON.parse(localStorage.getItem(key));
+    return storedValue != null ? storedValue : defaultValue;
+  } catch (e) {
```

Comment:
Changed this code to check the null value explicitly instead of using `||` operator, since it always evaluated to `true` when `defaultValue` was set to `true`.

Reply:
This makes a lot of sense for storing booleans! :100: